src/Fixer/PhpBasic/CodeStyle/MethodNamingFixer.php

```diff
@@ -17,10 +17,9 @@ final class MethodNamingFixer extends AbstractFixer
 {
     public const BOOL_FUNCTION_COMMENT = 'Question-type functions always return boolean (https://bit.ly/psg-methods)';
 
-    private array $boolFunctionPrefixes;
-
-    public function __construct()
-    {
+    public function __construct(
+        private array $boolFunctionPrefixes = [],
+    ) {
```

Comment:
Should fit into single line

Reply:
We need to decide about PHP version first. I will change this line depending on PHP version.

Reply:
Fixed